<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferencesDefaults</class>
 <widget class="QWidget" name="PreferencesDefaults">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>561</width>
    <height>558</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Lists and tables</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_1">
      <property name="margin">
       <number>5</number>
      </property>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Font</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFontComboBox" name="font"/>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Font size</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="fontsize"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Default file</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <property name="horizontalSpacing">
       <number>5</number>
      </property>
      <property name="verticalSpacing">
       <number>5</number>
      </property>
      <property name="margin">
       <number>5</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="defaultDiveLogFileLabel">
        <property name="text">
         <string>Default dive log file</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="defaultFileBehaviorLayout">
        <item>
         <widget class="QRadioButton" name="noDefaultFile">
          <property name="text">
           <string>&amp;No default file</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="localDefaultFile">
          <property name="text">
           <string>&amp;Local default file</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="cloudDefaultFile">
          <property name="text">
           <string>Clo&amp;ud storage default file</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Local dive log file</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLineEdit" name="defaultfilename"/>
        </item>
        <item>
         <widget class="QToolButton" name="btnUseDefaultFile">
          <property name="text">
           <string>Use default</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="chooseFile">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Display invalid</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="displayinvalid">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="title">
      <string>Default cylinder</string>
     </property>
     <layout class="QFormLayout" name="formLayout_6">
      <property name="horizontalSpacing">
       <number>5</number>
      </property>
      <property name="verticalSpacing">
       <number>5</number>
      </property>
      <property name="margin">
       <number>5</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Use default cylinder</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QComboBox" name="default_cylinder"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_7">
     <property name="title">
      <string>Animations</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="margin">
       <number>5</number>
      </property>
      <item>
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>Speed</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="velocitySlider">
        <property name="maximum">
         <number>500</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="velocitySpinBox">
        <property name="maximum">
         <number>500</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_10">
     <property name="title">
      <string>Video thumbnails</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="horizontalSpacing">
       <number>5</number>
      </property>
      <property name="verticalSpacing">
       <number>5</number>
      </property>
      <property name="margin">
       <number>5</number>
      </property>
      <item row="1" column="0">
       <widget class="QLabel" name="ffmpegExectuableLabel">
        <property name="text">
         <string>ffmpeg executable</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLineEdit" name="ffmpegExecutable"/>
        </item>
        <item>
         <widget class="QToolButton" name="ffmpegFile">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="videoThumbnailPositionLabel">
        <property name="text">
         <string>Extract at position</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSlider" name="videoThumbnailPosition">
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>20</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="extractVideoThumbnailsLabel">
        <property name="text">
         <string>Extract video thumbnails</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="extractVideoThumbnails">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_9">
     <property name="title">
      <string>Clear settings</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="resetRememberedDCs">
        <property name="text">
         <string>Reset remembered dive computers</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="resetSettings">
        <property name="text">
         <string>Reset all settings to their default value</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_extra_star_widgets">
     <property name="title">
      <string>Environmental parameters in Information tab</string>
     </property>
     <layout class="QHBoxLayout" name="extraWidgetLayout">
      <item>
       <widget class="QCheckBox" name="extraEnvironmentalDefault">
        <property name="text">
         <string>Show extended range of environmental paramaters</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_salinity">
     <property name="title">
      <string>Water salinity in Information tab</string>
     </property>
     <layout class="QHBoxLayout" name="salinityLayout">
      <item>
       <widget class="QCheckBox" name="salinityEditDefault">
        <property name="text">
         <string>Make water salinity editable</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>195</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnUseDefaultFile</sender>
   <signal>toggled(bool)</signal>
   <receiver>chooseFile</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>462</x>
     <y>136</y>
    </hint>
    <hint type="destinationlabel">
     <x>528</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>velocitySlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>velocitySpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>437</x>
     <y>299</y>
    </hint>
    <hint type="destinationlabel">
     <x>531</x>
     <y>301</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>velocitySpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>velocitySlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>547</x>
     <y>274</y>
    </hint>
    <hint type="destinationlabel">
     <x>482</x>
     <y>276</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
